src/compiler/clang.rs

```diff
@@ -210,6 +210,7 @@ counted_array!(pub static ARGS: [ArgInfo<gcc::ArgData>; _] = [
     take_arg!("-MT", OsString, CanBeSeparated, DepTarget),
     flag!("-Wno-unknown-cuda-version", PassThroughFlag),
     flag!("-Wno-unused-parameter", PassThroughFlag),
+    take_arg!("-Xarch", OsString, ConcatenatedAndSeparated('_'), PassThrough),
```

Comment:
It's actually unclear to me so far what `PassThrough` does after looking through the code.

Does it ignore the value passed, or does it bust the cache for every different value?